uk_bin_collection/uk_bin_collection/councils/MedwayCouncil.py

```
@@ -20,7 +20,23 @@ def parse_data(self, page: str, **kwargs) -> dict:
 
-        response = requests.get(api_url, verify=False)
+        # api.medway.gov.uk occasionally times out the first connection
+        # attempt but responds fine on retry. Do two short attempts rather
+        # than one long one so a flaky request still fits the production
+        # subprocess budget.
+        import time as _time
+        response = None
+        last_err = None
+        for attempt in range(2):
+            try:
+                response = requests.get(api_url, verify=False, timeout=15)
```

Ruff (0.15.9)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)